luminoth/detect.py

```diff
@@ -0,0 +1,40 @@
+import click
+import easydict
+import tensorflow as tf
+import os
+import yaml
+
+from luminoth.utils.predict import get_prediction
+from PIL import Image, ImageDraw
+
+
+@click.command(help='Detects objects in a image.')
+@click.argument('image-path')
+@click.option('--config-file')
+@click.option('--model-name', default='fasterrcnn')
```

Comment:
This should also work with image classification. /cc @vierja

Reply:
`model-name` should be `model-type` but I'm going to remove it as a CLI option in #91.